src/recipe/admin/src/runtime/components/ErrorPage.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div
+    v-if="!loading"
+    data-testid="error-page"
+    class="p-8"
+  >
+    <div class="pt-8 space-y-4">
+      <Tag
+        severity="danger"
+        :value="statusCode"
+        class="text-4xl"
+      />
+      <h1 class="text-6xl">
+        {{ errorInfo.title }}
+      </h1>
+      <div class="text-2xl">
+        {{ errorInfo.message }}
+      </div>
+      <div class="text-2xl">
+        Try the links from the menu below to view the page you want to access.
```

Comment:
use schema for this and below texts.

p.s. this is a temp solution until we implement localization